packages/fluentui/react-northstar/src/components/Attachment/Attachment.tsx

```diff
@@ -122,29 +124,17 @@ const Attachment: React.FC<WithAsProp<AttachmentProps>> &
 
   const element = (
     <ElementType {...getA11Props('root', { className: classes.root, onClick: handleClick, ...unhandledProps })}>
-      {Box.create(icon, {
-        defaultProps: () => ({ styles: resolvedStyles.icon }),
-      })}
+      {AttachmentIcon.create(icon)}
+
       {(header || description) && (
-        <div className={classes.content}>
-          {Text.create(header, {
-            defaultProps: () => ({ styles: resolvedStyles.header }),
-          })}
-
-          {Text.create(description, {
-            defaultProps: () => ({ styles: resolvedStyles.description }),
-          })}
+        <div className="ui-attachment__content">
```

Comment:
Just curious, why we don't have `AttachmentBody` component that will have the header and description props and we would create those inside that component?